### ISSUE TITLE
For the rows and columns of the input matrix

### ISSUE THREAD

**Issue by Apprentice2** [stekhoven/missForest#29]
Thank you for developing this wonderful package called missForest. I am considering applying missForest to proteome data consisting of n=20 samples.  This data has a maximum of  p proteins in each sample, and is p>>n high dimensional data.
I know that missForest can cope with high dimensionality (p ≫ n).　I would like to confirm one point regarding the use of missForest. According to the manual, I can do this by applying the matrix x to the missForest function, where each column of x corresponds to each variable and each row of x corresponds to each observation (each sample).
Would this be the same for high dimensional data where x is p>>n?
In other words, for my data, do I need to apply data into the missForest function where each column of x corresponds to each protein and each row of x corresponds to each sample? I have seen examples in other studies where each column of x corresponds to each sample and each row of x corresponds to each protein, and I have seen cases where the data corresponding to each protein is entered into the missForest function, but is this process inappropriate?
I would be grateful if you could enlighten me.

**Comment by stekhoven** [comment]
Yes, it would be the same, i.e. rows are observations/subjects/participants (in your case the 20 samples) and columns are variables/measurements/values from these samples (in your case the proteins).

**Comment by Apprentice2** [comment]
Thank you very much. I could understand very well.
